src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -307,7 +315,9 @@ def as_text(self):
 
         # Always add body as it's been cleaned up
         if self.body:
-            body += '\n```\n{}\n```'.format(self.body)
+            body += '\n{}'.format(self.body)
```

Comment:
Nit: Here, you're accidentally reverting a change that I did recently. That's why the tests fail (if you look at the failed test logs, you'll see that it complains about missing "```" in the output).